core/src/main/resources/swagger/data-repo-only.yaml

```diff
@@ -165,27 +137,59 @@ paths:
       summary: Delete a snapshot
       operationId: deleteDataRepoSnapshot
       parameters:
-        - in: path
-          description: The workspace namespace
-          name: namespace
-          required: true
-          type: string
-        - in: path
-          description: The workspace name
-          name: name
-          required: true
-          type: string
-        - in: path
-          description: The snapshot ID
-          name: snapshotId
-          required: true
-          type: string
+        - $ref: '#/parameters/namespacePathParam'
+        - $ref: '#/parameters/namePathParam'
+        - $ref: '#/parameters/snapshotIdPathParam'
       security:
         - authorization:
             - openid
             - email
             - profile
 
+parameters:
+  billingProjectQueryParam:
+    in: query
+    description: Billing project to use for queries to the data reference
+    name: billingProject
+    required: false
+    type: string
+  dataReferenceQueryParam:
+    in: query
+    description: Data reference name to query for entity metadata
+    name: dataReference
+    required: false
+    type: string
+  namespacePathParam:
+    in: path
+    description: The workspace namespace
+    name: namespace
+    required: true
+    type: string
+  namePathParam:
+    in: path
+    description: The workspace name
+    name: name
+    required: true
+    type: string
+  snapshotIdPathParam:
+    in: path
+    description: The snapshot ID
+    name: snapshotId
+    required: true
+    type: string
+  workspaceNamespacePathParam:
```

Comment:
you have both a `workspaceNamespacePathParam` and a `namespacePathParam` defined on line 162.

Reply:
I had two because I just extracted them from the existing API definitions - but you're right, and I consolidated them by changing the API definitions (in swagger only)